### PR TITLE
Create williamsmi Grapevine Tx recommendation file

### PR DESCRIPTION
**Pull request by williamsmi**
- This is a bullet point.

- [ ] This is a task list item.

### CODE REVIEW

**Comment by rzuckerm** [comment]
This is a comment

File: williamsmi-grapevine.tx.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Grapevine Tx
```
Comment (rzuckerm) [inline williamsmi-grapevine.tx.md:1]: Good content.

I have no idea where Grapevine, TX is.

Comment (williamsmi) [reply williamsmi-grapevine.tx.md:1]: It's smack in the middle between Dallas and Fort Worth, Tx

**Review by rzuckerm — APPROVED** [review]
:+1:

**Pull request merged**